pkg/localplan/watcher.go

```diff
@@ -70,7 +70,7 @@ func (w *watcher) listFiles(ctx context.Context, force bool) error {
 	return nil
 }
 
-func (w *watcher) listFilesIn(ctx context.Context, base string, force bool) error {
+func (w *watcher) listFilesIn(ctx context.Context, base string, _ bool) error {
```

Comment:
why is this being introduced in a migration PR?

Reply:
the reason is the same as the [above](https://github.com/rancher/system-agent/pull/170#discussion_r1661374121), to make the revive happy.